.github/workflows/web-service.yml

```
@@ -2,12 +2,20 @@ name: Web Service Release
 
 on:
   workflow_dispatch:
+    inputs:
+      data_json_url:
+        required: true
+        description: A json file url with the data needed for the service (i.e. pastebin.pl)
+      tag:
+        required: true
+        description: a tag that will be used for identifying this version (xaynetci/xayn_discovery_web_service:$TAG)
+
 
 permissions:
   contents: read
 
 env:
-  DENY_WARNINGS: false
+  DENY_WARNINGS: false  
```

🔧 pls don't add needless whitespace
```suggestion
  DENY_WARNINGS: false
```